src/aleph_client/commands/program.py

```diff
@@ -83,8 +84,36 @@ def upload(
         help="Hash of the runtime to use for your program. Defaults to aleph debian with Python3.8 and node. You can also create your own runtime and pin it",
     ),
     beta: bool = typer.Option(False),
+
+
+
     debug: bool = False,
     persistent: bool = False,
+    persistent_volumes: Optional[Tuple[str, str, str]] = typer.Option(
+        (None, None, None),
+        help= '''Takes 3 parameters                                                                  
+        A persistent volume is allocated on the host machine at any time
+        eg:host my-volume 1 '''),
+
+    ephemeral_volumes: Optional[List[str]] = typer.Option(
+        None,
+        help=
+        '''Takes 1 parameters 
+        Ephemeral volumes can move and be removed by the host,
+        Garbage collected basically, when the VM isn't running  eg: 100 '''),
+
+    immutable_volumes: Optional[Tuple[str, bool,str]] = typer.Option(
+        (None, None, None),
+        help=
+        '''Takes 3 parameters
+         Immutable volume is one whose contents do not change 
+        eg:25a393222692c2f73489dc6710ae87605a96742ceef7b91de4d7ec34bb688d94  True/False   mount=/mnt/volume
```

Comment:
```suggestion
        eg: --immutable_volumes 25a393222692c2f73489dc6710ae87605a96742ceef7b91de4d7ec34bb688d94 True /mnt/volume
```
Consistent style, add the parameter flag, the example should be copy-pasteable for users